site/frontend/src/pages/compare/shared.ts

```diff
@@ -78,3 +78,18 @@ export function getBoolOrDefault(
   }
   return defaultValue;
 }
+
+export function benchmarkNameMatchesFilter(
+  benchmarkName: string,
+  filterName: string | null
+): boolean {
+  if (!filterName) return true;
```

Comment:
Now this should be changed to something like this:
```typescript
if (filterName === null) return true;
const trimmedFilterName = filterName.trim();
if (trimmedFilterName === "") return true;
```
So that a name filter containing only whitespace will match all benchmarks.

Reply:
Of course I can add it explicitly, it's already doing same thing:
(empty regex can match everything)

![CleanShot 2024-06-09 at 21 34 19@2x](https://github.com/rust-lang/rustc-perf/assets/53410646/5f0063db-0032-4e72-93a7-e07a46e45d6c)